android-components/components/lib/crash-sentry/src/main/java/mozilla/components/lib/crash/sentry/SentryService.kt

```diff
@@ -189,6 +210,9 @@ class SentryService(
     }
 }
 
+@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+internal const val UNCAUGHT_EXCEPTION_TYPE = "UncaughtExceptionHandler"
```

Comment:
Is this supposed to be used in test? 🤔 Don't think it's used right now.

Reply:
good catch! I was going to assert on this as well but lost track.